src/cargo/sources/git/known_hosts.rs

```diff
@@ -588,7 +589,14 @@ impl KnownHost {
         }
         for pattern in self.patterns.split(',') {
             let pattern = pattern.to_lowercase();
-            // FIXME: support * and ? wildcards
```

Comment:
Was under the impression that this feature is tracked in <https://github.com/rust-lang/cargo/issues/11577>, but apparently not.

It seems like OpenSSH [supports only `*` and `?` wildcards](https://man.openbsd.org/sshd.8#SSH_KNOWN_HOSTS_FILE_FORMAT). One caveat of using `glob` here is we provide more than we should. Anyway, it is good to catch up with the common format.